src/pudl/metadata/fields.py

```diff
@@ -1874,6 +2009,80 @@
             "min, equal to or less than 5 min, or some other way."
         ),
     },
+    "monthly_average_consumption_rate_gallons_per_minute": {
+        "description": "Monthly average consumption rate of cooling water",
+        "type": "number",
+        "unit": "gpm",
+    },
+    "monthly_average_discharge_rate_gallons_per_minute": {
+        "description": "Monthly average discharge rate of cooling water - available 2010+",
+        "type": "number",
+        "unit": "gpm",
+    },
+    "monthly_average_discharge_temperature_fahrenheit": {
+        "description": "Average cooling water temperature at discharge point",
+        "type": "number",
+        "unit": "F",
+    },
+    "monthly_average_diversion_rate_gallons_per_minute": {
+        "description": "Monthly average diversion rate of cooling water",
+        "type": "number",
+        "unit": "gpm",
+    },
+    "monthly_average_intake_temperature_fahrenheit": {
+        "description": "Average cooling water temperature at intake point",
+        "type": "number",
+        "unit": "F",
+    },
+    "monthly_average_withdrawal_rate_gallons_per_minute": {
+        "description": "Monthly average withdrawal rate of cooling water",
+        "type": "number",
+        "unit": "gpm",
+    },
+    "monthly_maximum_discharge_temperature_fahrenheit": {
+        "description": "Maximum cooling water temperature at discharge",
+        "type": "number",
+        "unit": "F",
+    },
+    "monthly_maximum_intake_temperature_fahrenheit": {
+        "description": "Maximum cooling water temperature at intake",
+        "type": "number",
+        "unit": "F",
+    },
+    "monthly_total_chlorine_lbs": {
+        "description": (
+            "Amount of elemental chlorine added to cooling water monthly. "
+            "May be just the amount of chlorine-containing compound if "
+            "schedule 9 is filled out."
+        ),
+        "type": "number",
+        "unit": "lb",
+    },
+    "monthly_total_consumption_volume_gallons": {
+        "description": "Monthly volume of water consumed at consumption point (accurate to 0.1 million gal)",
+        "type": "number",
+        "unit": "gal",
+    },
+    "monthly_total_cooling_hours_in_service": {
```

Comment:
We use "hr" elsewhere and should stick to that for consistency

Reply:
Hmm, I see `operating_time_hours` but I don't see `_hr` anywhere in the field names; in codes I see a lot more `_hour` than i see `_hr`.

Reply:
I meant in the units of `operating_time_hours`, I think the unit listed is `hr`

Reply:
Ahhh good eye thanks!